Purchases/NSData+RCExtensions.m

```diff
@@ -8,13 +8,6 @@
 NS_ASSUME_NONNULL_BEGIN
 
 
-@interface NSData (RCExtensions)
```

Comment:
wasn't needed and was actually duplicated. I think there was a comment about it and I missed it